airbyte-webapp/src/components/CreateConnection/CreateConnectionForm.tsx

```diff
@@ -101,9 +107,8 @@ const CreateConnectionFormInner: React.FC<CreateConnectionPropsInner> = ({ schem
       <div className={styles.connectionFormContainer}>
         <Formik
           initialValues={initialValues}
-          validationSchema={connectionValidationSchema(mode)}
+          validationSchema={createConnectionValidationSchema({ mode, allowSubOneHourCronExpressions })}
           onSubmit={onFormSubmit}
-          validateOnChange={false}
```

Comment:
I don't think we can get around removing `validateOnChange={false}`, but I'm not exactly sure why this was needed in the first place. Maybe @tealjulia can shed some light?

Reply:
https://www.loom.com/share/eb2c9aff23e4407bb3af1ce7e09a13a9

Here's an example.  Previous to adding this line, the validation would fire for the entire form if any piece of the connection object was updated.  Sometimes, this meant that validation would fire even for rows that a user hadn't interacted with yet, which seemed pretty unexpected from a user perspective.

Reply:
Thanks for the loom! Now I see what the issue is. I think we need to dig into the validation of `CatalogSection` and figure out why it triggers when an unrelated `CatalogSection` is touched. Probably the error state is being triggered regardless of the `field.touched` flag.

I filed a bug [here](https://github.com/airbytehq/airbyte/issues/18176) (there wasn't an issue for that yet, right?) I'll have a look tomorrow and see if it's feasible to fix before this PR is merged.